src/app/pages/dashboard/dashboard.page.ts

```diff
@@ -286,7 +286,7 @@ export class DashboardPage implements OnInit {
       repayment_cycle_id: ["", Validators.required],
       business_type_id: ["", Validators.required],
       payment_type_id: ["", Validators.required],
-      custom_date: ["", Validators.required]
+      custom_date: [""]

```

Comment:
If i select custom and forget to enter it, it will break from backend, validation should still be applied conditionally though